configure.in

```diff
@@ -7047,6 +7048,11 @@ if test "$wxUSE_WEBVIEW" = "yes"; then
     USE_WEBVIEW=1
     AC_DEFINE(wxUSE_WEBVIEW)
     SAMPLES_SUBDIRS="$SAMPLES_SUBDIRS webview"
+    if test "$wxUSE_WEBVIEW_CHROMIUM" = "yes"; then
+        USE_WEBVIEW_CHROMIUM=1
```

Comment:
This doesn't seem right, the option is "yes" by default (not sure if it should be, if not, `DEFAULT_wxUSE_WEBVIEW_CHROMIUM` should be set to "no") and we don't check that we have the required headers/libraries here, so it looks like the compilation will fail out of the box.

We should have `AC_TRY_LINK` check here.